apps/package.json

```diff
@@ -58,6 +58,7 @@
     "@storybook/react-webpack5": "~8.1.11",
     "@testing-library/jest-dom": "^6.4.8",
     "@testing-library/react": "^12.1.5",
+    "@testing-library/react-hooks": "^8.0.1",
```

Comment:
@lfryemason Just a heads up that we're moving in this direction in the acq world! @drizco for context- Liam has been leading the charge toward React Testing Library, so hoping to keep him in the loop as much as possible:)

Reply:
Are we able to use `renderHooks` from `@testing-library/react` instead of the standalone library? [It seems like they recently added it](https://docs.google.com/document/d/1o_7Ii02DxzV9D9HvL3-XAs0tJAZUCce2zsJy8AhIQUk/edit?pli=1)

Reply:
Is that the correct link @lfryemason? I'm taken to a tech spec.

Reply:
It absolutely is not: https://github.com/testing-library/react-hooks-testing-library?tab=readme-ov-file#a-note-about-react-18-support

Reply:
it's not in our current version of react testing library, looks like we're on 12.1.5 

I can try updating though

Reply:
it seems like bumping to 13.x to get `renderHook` doesn't work for react versions under 18 unfortunately. are we considering a react update to 18 soon?